.github/workflows/build-test.yaml

```diff
@@ -154,6 +154,27 @@ jobs:
           push: true
 
 
+  notify_slack_test:
```

Comment:
i dont think that this is needed in the build-test workflow. only nightly

Reply:
i'm aware, see my PR description above.